.github/workflows/build.yaml

```
@@ -0,0 +1,76 @@
+---
+name: Build Containers
+
+"on":
+  pull_request:
+
+env:
+  KICS_user_is_root: 67fd0c4a-68cf-46d7-8c41-bc9fba7e40ae
+  KICS_apt_version_pin: 965a08d7-ef86-4f14-8792-4a3b2098937e
+
+jobs:
+  kics-scan:
+    name: KICS Scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: "repo"
+      - name: KICS Github Action
+        uses: checkmarx/kics-action@v1.5
+        with:
+          path: "repo"
+          output_path: "results"
+          exclude_queries: ${{ env.KICS_user_is_root }},${{ env.KICS_apt_version_pin }}
+          fail_on: high,medium
+          enable_comments: true
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    needs:
+      - kics-scan
+    strategy:
+      matrix:
+        terraform_version:
+          - 0.14.7
+          - 1.2.4
+          - 1.2.5
+        ruby_version:
+          - 3.1.2
+    steps:
+      - name: Code checkout
+        uses: actions/checkout@v2
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
```

I've previously used `actor` here, which is the example provided in the Action. I'm not sure what difference it really makes
```suggestion
          username: ${{ github.actor }}
```

---

For future reference (for me)

actor: 
```text
The username of the user that triggered the initial workflow run. If the workflow run is a re-run, this value may differ from github.triggering_actor. Any workflow re-runs will use the privileges of github.actor, even if the actor initiating the re-run (github.triggering_actor) has different privileges.
```

owner:
```text
The repository owner's name. For example, Codertocat.
```

I think owner is right in this context?

---

happy to see what happens at this stage